src/snowflake/cli/_plugins/nativeapp/artifacts.py

```diff
@@ -658,6 +667,9 @@ def build_bundle(
     for artifact in artifacts:
         bundle_map.add(artifact)
 
+    if bundle_map.is_empty():
+        raise ArtifactError("No artifacts were found, nothing to do.")
```

Comment:
Or I can create a new error class for specificity - e.g. `NoArtifactError`, if we prefer that?

Reply:
the error type is fine by me, but I think the message can be a bit more specific
```suggestion
        raise ArtifactError("No artifacts mapping found in project definition, nothing to do.")
```